inst/scripts/test_auto_tweet.R

```diff
@@ -20,7 +20,7 @@ is_tweet_length <- function(.x, n = 280) {
 assignInNamespace("is_tweet_length", is_tweet_length, ns = "rtweet")
 
 on_error_email_to(c(gmail("malcolmbarrett")))
-safe_action_auto_tweet <- action_safely(action_auto_tweet)
+# if testing post_tweet_of_type manually on a weekend, set apply_test_dat = TRUE
+# in action_auto_tweet() below
+safe_action_auto_tweet <- action_safely(action_auto_tweet(twitter_token = sandbox_token()))
```

Comment:
I've made this update to the code, but I end up being able to run a test because I don't have the SER email account's secret. If I remember, I think that might be why I did it the original way.

Can you share that info with me?